### ISSUE TITLE
Fix device_status when turning on/off fans

### ISSUE THREAD

**Issue by kevchu3** [webdjoe/pyvesync#116]
Fans of class `VeSyncAir200S`, `VeSyncAir300S400S`, and `VeSyncHumid200300S` don't currently set `self.device_status` in the `turn_off()` and `turn_on()` methods (it was only implemented in `VeSyncAir131`).

This leads to inconsistencies when querying `self.device_status` to determine a fan's current state.  In Home Assistant's core implementation of [fan.py](https://github.com/home-assistant/core/blob/dev/homeassistant/components/vesync/fan.py), this unexpectedly leads to the fan device seeming to be stuck in whatever state it was initialized in, even though the physical fan actually turns on and off.

Here's a sanity check.  You can see that this humidifier starts in the 'on' state but `device_status` doesn't change state even after `turn_off()` is invoked:

```
$ python3
Python 3.10.2 (main, Jan 17 2022, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyvesync import VeSync
>>> manager = VeSync("USERNAME", "PASSWORD", "US")
>>> manager.login()
True
>>> manager.update()
>>> humidifier = manager.fans[0]
>>> humidifier.device_status
'on'
>>> humidifier.turn_off()
True
>>> humidifier.device_status
'on'
```

**Comment by webdjoe** [comment]
@kevchu3 Sorry for delay refactored the library to make it easier to configure devices - should be fixed here if you wouldn't mind testing - https://github.com/webdjoe/pyvesync/tree/air-refactor

**Comment by webdjoe** [comment]
fixed in #121

**Comment by kevchu3** [comment]
> @kevchu3 Sorry for delay refactored the library to make it easier to configure devices - should be fixed here if you wouldn't mind testing - https://github.com/webdjoe/pyvesync/tree/air-refactor

@webdjoe I've tested and put some comments here: https://github.com/webdjoe/pyvesync/pull/121#issuecomment-1064712271
